pages/api/docs/getAllRepos.ts

```diff
@@ -3,22 +3,31 @@ import { getServerSession } from 'next-auth/next';
 import { RepoIdentifier } from '../../../types/repository';
 import { getUserRepositories } from "../../../utils/providerAPI/getUserRepositories";
 import { authOptions } from '../auth/[...nextauth]';
+import rudderStackEvents from '../events';
 
 const getUserRepositoriesHandler = async (req: NextApiRequest, res: NextApiResponse) => {
 	const session = await getServerSession(req, res, authOptions);
 	if (!session) {
+		const eventProperties = { response_status: 401 };
+        rudderStackEvents.track("absent", "", 'docs-user-repos-handler', { type: 'HTTP-401', eventStatusFlag: 0, eventProperties });
```

Comment:
```suggestion
        rudderStackEvents.track("absent", "", 'user-repos-from-providers', { type: 'HTTP-401', eventStatusFlag: 0, eventProperties });
```

I don't think the docs prefix is very useful here.